spec/controllers/application_controller/explorer_spec.rb

```diff
@@ -40,7 +40,8 @@
         ems = FactoryGirl.create(:ems_vmware, :ems_folders => [ems_folder])
 
         user = FactoryGirl.create(:user_admin)
-        user.current_group.set_managed_filters([["/managed/service_level/gold"]])
+        user.current_group.entitlement = Entitlement.create!(:filters => {'managed'   => [["/managed/service_level/gold"]],
+                                                                          'belongsto' => []})
```

Comment:
I guess I was thinking about changing the format of this column eventually.  I thought hardcoded hash keys littered all over the place would be harder to find/replace than a single place in the set_(managed|belongsto)_filter method.  I don't know though.